Day04/indecisive/src/components/CardComponent/CardComponent.js

```diff
@@ -0,0 +1,115 @@
+        id: "meenakshipandey#1422"
+    },
+    {
+        photo: "https://drive.google.com/file/d/16n4Tjzb7yAMD4G4NxPMXEIAmtmO5fsLa/view?usp=sharing",
+        name: "Sudhanshu Kansal",
+        place: "Ghaziabad",
+        college: "Lovely Professional University ",
+        designation: "Aspiring Software Engineer ",
+        linkedinProfile: "https://www.linkedin.com/in/sudhanshukansal09/",
+        id: "sudhanshukansal#6088"
+    },
+    {
+        photo: "https://scontent.fdel27-5.fna.fbcdn.net/v/t1.6435-9/120300090_1401917953339744_4526160489616159108_n.jpg?_nc_cat=110&ccb=1-7&_nc_sid=09cbfe&_nc_ohc=vf01-5emDmEAX-A_b6P&tn=cOTsm3FbCjEH4WMU&_nc_ht=scontent.fdel27-5.fna&oh=00_AfA-HYnJL6dh8lVSiv1kfJyVhldGgo-Jz4JVVohCYD3V_g&oe=63A84959",
+        name: "Meenakshi Pandey",
+        place: "Electronic City, Bangalore ",
+        company: "Siemens Healthineers",
+        designation: "Full Stack Developer",
+        id: "meenakshipandey#1422",
+        linkedinProfile: "https://www.linkedin.com/in/meenakshi-pandey-590b3079/"
+    }, {
+        photo: "https://scontent.fdel27-5.fna.fbcdn.net/v/t1.6435-9/120300090_1401917953339744_4526160489616159108_n.jpg?_nc_cat=110&ccb=1-7&_nc_sid=09cbfe&_nc_ohc=vf01-5emDmEAX-A_b6P&tn=cOTsm3FbCjEH4WMU&_nc_ht=scontent.fdel27-5.fna&oh=00_AfA-HYnJL6dh8lVSiv1kfJyVhldGgo-Jz4JVVohCYD3V_g&oe=63A84959",
+        name: "Meenakshi Pandey",
+        place: "Electronic City, Bangalore ",
+        company: "Siemens Healthineers",
+        designation: "Full Stack Developer",
+        id: "meenakshipandey#1422",
+        linkedinProfile: "https://www.linkedin.com/in/meenakshi-pandey-590b3079/"
+    }
+]
+
+const CardComponent = () => {
+
+    return (
+        <>
+            {teamDetails.map((data, index) => (
+                <div className="card">
+                    <img className="card-img" src={data.photo} alt='Image Not found' onError={({ currentTarget }) => {
+                        currentTarget.onerror = null; // prevents looping
+                        currentTarget.src = require("../../assets/person.png");
+                    }} />
+                    <h1><b>{data.name}</b></h1>
+                    <p className="title"><b>{data.designation}</b></p>
+                    <div className="card-info">
+                        <i>
+                            <p>{data.company}</p>
+                            <span className="card-location">
+                                <img src={require("../../assets/location.jpg")} target="_blank" />
+                                {data.place}
+                            </span>
+                            <p>{data.id}</p>
+
+                            <span className="linkedin">
+                                <a href={data.linkedinProfile} target="_blank">
+                                    <img className="linkedin-img" src={require("../../assets/linkedin.png")} />
+                                </a >

```

Comment:
You don't need require here. Can use ES modules import.
Appreciate the handling on onError.